releasenotes/notes/rag-prompt-context-vars-f6cb63a4903f9337.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    LLM Observability: Adds `rag_context_variables` and `rag_query_variables` helper fields to the `Prompt`
```

Comment:
not sure if we want to make this so public just yet -- meaning we'll have to support it going forward for the foreseeable future. If we are just looking to trial this with some specific customers then I'd hold off on the release note.

Reply:
I went a bit back and forth on this one; but I agree we should hold off on the release note since OOTB hallucination metric on our backend is not ready yet